Godeps/Godeps.json

```diff
@@ -843,12 +844,12 @@
 		},
 		{
 			"ImportPath": "github.com/docker/distribution/digest",
-			"Comment": "v2.4.0-rc.1-38-gcd27f17",
+			"Comment": "v2.4.0-rc.1-38-gcd27f179",
```

Comment:
This is the other reason to run godep in containers.. different git versions provided slightly different strings 😞 . Is this your local env, or inside a container?

Reply:
local.  The problem with doing this in containers is that some envs (jenkins) are already containerized, and we end up with a tangle of "is this already in a container?  call a different script" sort of hacks.  I want to move this forward as "it works locally, where locally can be in-a-container or not" as the general MO.  Running this inside a container requires YET ANOTHER script to avoid rsyncing over the state in between steps.